apps/frontend/src/app/modules/misc/components/colleagues/colleagues.component.css

```diff
@@ -87,14 +87,15 @@
   .card-title {
     grid-column: span 1;
   }
-}
 
-@media screen and (max-width: 576px) {
+  .sidebar {
+    display: none;
+  }
+
   .container {
     grid-template-columns: 1fr;
   }
+}
 
-  .sidebar {
-    display: none;
-  }
+@media screen and (max-width: 576px) {
```

Comment:
Do not leave unused media queries in the code please.

Reply:
Removed unused query in e874475